docs/LwM2M_Client/STMicroelectronics/B-L475E-IOT01A.md

```diff
@@ -9,35 +9,35 @@ Integrate your B-L475E-IOT01A Discovery kit board.
 - Installed **ST-Link** or **OpenOCD** debugger.
 - A user with access to the Coiote IoT Device Management platform and appropriate permissions.
 
-## Prepare binaries
+## Prepare Anjay client application
 ### Use an already built binary
 
-To get the latest binary file and flash the board:
+To get the latest binary file and flash it to the board:
 
 0. Go to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
 0. Download the `demo_B-L475E-IOT01A1_merged.bin` file.
 0. To flash the board, open your **File manager** and drag the downloaded `.bin` file to your **DIS_L4IOT** external device.
 0. You will see a blinking diode on your board. The diode will stop blinking as soon as the flashing is finished.
 
-The board is now flashed: you can jump to the [Connecting to the LwM2M Server](#connecting-to-the-lwm2m-server) section.
+The board is now flashed: you can jump to the [Connect to the LwM2M Server](#connect-to-the-lwm2m-server) section.
```

Comment:
I don't like the ":" sign in this sentence. Also, if the steps are the same as in STM32L496G-DISCO/BG96 and B-L462E-CELL1/TYPE1SC why only in this tutorial do you use this sentence?